2025-01-06 09:11:24,028 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:11:32,407 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:12:26,333 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:13:46,345 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:18:49,821 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:22:40,572 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:36:04,463 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:44:50,534 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-06 09:46:16,454 ERROR frappe Could not take error snapshot: characters_written
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 97, in application
    response = frappe.api.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/api.py", line 55, in handle
    return frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1619, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 73, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 84, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 44, in get_meta
    meta = FormMeta(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 55, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 65, in load_assets
    self.add_code()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 120, in add_code
    self.add_html_templates(path)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 135, in add_html_templates
    for fname in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/frappe/frappe-bench/apps/library_management/library_management/library_management/doctype/article'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 60, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/error.py", line 153, in get_snapshot
    value = pydoc.text.repr(getattr(evalue, name))
AttributeError: characters_written
2025-01-08 06:46:10,233 ERROR frappe New Exception collected with id: 2025-01-08 06:46:09.568053-127.0.0.1-572
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '2025-01-07 16:50:06.810147', 'cmd': 'frappe.desk.form.load.getdoctype'}
2025-01-08 06:46:21,034 ERROR frappe New Exception collected with id: 2025-01-08 06:46:20.879936-127.0.0.1-4f2
Site: demo
Form Dict: {'doctype': 'Article', 'with_parent': '1', 'cached_timestamp': '2025-01-07 16:50:06.810147', 'cmd': 'frappe.desk.form.load.getdoctype'}
2025-01-08 06:46:52,669 ERROR frappe New Exception collected with id: 2025-01-08 06:46:52.618165-127.0.0.1-b4c
Site: demo
Form Dict: {'doctype': 'Article', 'name': '1984', 'cmd': 'frappe.desk.form.load.getdoc'}
2025-01-08 06:46:59,361 ERROR frappe New Exception collected with id: 2025-01-08 06:46:59.318583-127.0.0.1-3ce
Site: demo
Form Dict: {'doctype': 'Article', 'name': 'The Gift', 'cmd': 'frappe.desk.form.load.getdoc'}
2025-01-09 04:44:11,250 ERROR frappe New Exception collected with id: 2025-01-09 04:44:10.988445-127.0.0.1-24d
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:15,003 ERROR frappe New Exception collected with id: 2025-01-09 04:44:14.954039-127.0.0.1-d8f
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:18,704 ERROR frappe New Exception collected with id: 2025-01-09 04:44:18.674781-127.0.0.1-019
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:25,753 ERROR frappe New Exception collected with id: 2025-01-09 04:44:25.708355-127.0.0.1-919
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:26,825 ERROR frappe New Exception collected with id: 2025-01-09 04:44:26.792632-127.0.0.1-860
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:27,357 ERROR frappe New Exception collected with id: 2025-01-09 04:44:27.304343-127.0.0.1-480
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:27,958 ERROR frappe New Exception collected with id: 2025-01-09 04:44:27.931977-127.0.0.1-108
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:44:28,626 ERROR frappe New Exception collected with id: 2025-01-09 04:44:28.587955-127.0.0.1-6b1
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:55:59,268 ERROR frappe New Exception collected with id: 2025-01-09 04:55:58.869872-127.0.0.1-03b
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:56:03,198 ERROR frappe New Exception collected with id: 2025-01-09 04:56:03.195359-127.0.0.1-c94
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:56:32,438 ERROR frappe New Exception collected with id: 2025-01-09 04:56:32.113458-127.0.0.1-324
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
2025-01-09 04:56:33,848 ERROR frappe New Exception collected with id: 2025-01-09 04:56:33.482026-127.0.0.1-547
Site: demo
Form Dict: {'cmd': 'login', 'usr': 'Administrator', 'device': 'desktop'}
